dependencies/dependencies.json

```diff
@@ -0,0 +1,26 @@
+[
+    {
+        "library": "boost",
+        "version": "1.81.0"
+    },
+    {
+        "library": "maeparser",
+        "version": "1.3.1"
+    },
+    {
+        "library": "fmt",
+        "version": "6.1.2"
+    },
+    {
+        "library": "qt6",
+        "version": "6.2.5"
+    },
+    {
+        "library": "rdkit",
+        "version": "master"
```

Comment:
are we on master? Or is a specific hash we should be using?
https://opengrok.schrodinger.com/xref/mmshare/build_tools/library_definitions.json?r=9c596363#318